src/main/java/subway/path/domain/Path.java

```diff
@@ -0,0 +1,75 @@
+package subway.path.domain;
+
+import java.util.List;
+
+import org.jgrapht.GraphPath;
+import subway.path.exceptions.InvalidPathException;
+import subway.station.domain.Station;
+
+public class Path {
+    private final int BASE_FARE = 1_250;
+    private final int NON_DISCOUNTABLE_BASE_FARE = 350;
+    private final int FARE_PER_DISTANCE = 100;
+    private final int MIDDLE_DISTANCE_LIMIT = 10;
+    private final int FAR_DISTANCE_LIMIT = 50;
+    private final double FARE_INCREASE_MIDDLE_DISTANCE = 5.0d;
+    private final double FARE_INCREASE_FAR_DISTANCE = 8.0d;
+
+    private final GraphPath<Station, SubwayEdge> path;
+
+    public Path(GraphPath<Station, SubwayEdge> shortestPath) {
+        validatePath(shortestPath);
+        this.path = shortestPath;
+    }
+
+    private void validatePath(GraphPath<Station, SubwayEdge> shortestPath) {
+        if(shortestPath == null) {
+            throw new InvalidPathException();
+        }
+    }
+
+    public List<Station> getStations() {
+        return path.getVertexList();
+    }
+
+    public Integer getDistance() {
+        return (int) path.getWeight();
+    }
+
+    public int getFare(int age) {
+        int fare = BASE_FARE + getMaxLineExtraFare() + getDistanceExtraFare();
+        AgeGroup ageGroup = AgeGroup.getAgeGroup(age);
+        if(ageGroup.equals(AgeGroup.TODDLER)) {
```

Comment:
```suggestion
        if(ageGroup == AgeGroup.TODDLER) {
```
`enum`의 instance는 항상 singleton 이기 때문에 reference 비교만으로도 충분히 equality가 보장됩니다

Reply:
reference 비교로 수정했습니다